### DISCUSSION TITLE
indicators and backlight customisation

### DISCUSSION THREAD

**Discussion by Rend0e** [linux-is-awesome/gnome_extension_rocketbar#19] [Ideas]
Hi, for the backlight could there be an option to increase its brightness to make it more visible (similar to dash to panel). Also perhaps an option to change its shape to square.
![image](https://user-images.githubusercontent.com/20853575/183030553-93592d41-c7b0-4d91-a50a-31098ea427b2.png)
^ rocketbar (left) dash to panel (right), 

For the indicators could there be more shapes to customize, specifically 'solid' and  'dashes' from dash to panel, useful to differentiate active and inactive by giving different shapes etc. 
![image](https://user-images.githubusercontent.com/20853575/183031294-499287a4-bba3-4be4-9544-a60d9750ee36.png)
^ 'dashes' (right)
![image](https://user-images.githubusercontent.com/20853575/183031358-6bed5086-1246-4bfe-a677-ca17cd5b12b4.png)
^ 'solid' (right)

**Comment by ChepKun** [comment]
Hey!
I'm going to implement custom color selection for the backlight, it's on my todo list for the next release.
Right now there is a dominant color backlight option as an alternative.
I don't have plans to implement more indicator shapes for now, but probably later.

**Comment by ChepKun** [comment]
@Rend0e forgot to mention about the shape of the backlight. There is a Roundness option in Settings for app buttons. Is this what you are looking for?
![image](https://user-images.githubusercontent.com/5459599/183065732-d6817047-eb9b-4d5d-93df-efdf3cb6a13f.png)

**Reply by Rend0e** [reply]
Yep that works, it was for to pair with 'solid' indicators.
